### PR TITLE
We get many issues and no PR.

### PR DESCRIPTION
**Pull request by jpic**

### CODE REVIEW

**Comment by jpic** [comment]
If I could have your input on this before merging, wording could probably be improved, else I'll merge it as-is.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- clear steps to reproduce the bug,
+- a fork of the repo with an additionnal app in test_project to reproduce your
+  issue out-of-the-box,
+
+Do **not** bother:
+
+- pasting partial code in the issue text, ie. only forms and admin without
+  models, etc, etc... Actually don't bother posting code at all: fork the repo,
+  add example app, make a PR. We're not supporting anything that has not an
```
Comment (blueyed) [inline CONTRIBUTING.md]: "add an example app"

"We are more likely to fix/support something with an example app."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  example app.
+- I'm most probably **NOT** going to create the example app for you and support
+  the feature you need and do all the work by myself for free. So you need to:
+  fork the repo, add your example app, ensure that it's easy to run, and make a
+  PR. This app belongs to everybody who wants to "work it". If you don't want
```
Comment (blueyed) [inline CONTRIBUTING.md]: s/work it/make it work/

**Comment by jpic** [comment]
There's got to be a better way to solve this problem ...

**Comment by blueyed** [comment]
@jpic 
You gave up on that?

**Comment by jpic** [comment]
Well we're going to have the contributing guidelines from django-cc aren't
we ?

**Comment by blueyed** [comment]
True.

**Pull request closed without merging**